nomad/structs/acl.go

```diff
@@ -232,6 +233,27 @@ func (a *ACLToken) IsExpired(t time.Time) bool {
 	return a.ExpirationTime.Before(t) || t.IsZero()
 }
 
+// RoleSubset checks if a given set of role IDs are assigned to the ACL token.
+func (a *ACLToken) RoleSubset(roleIDs []string) bool {
```

Comment:
This might make the signature more obvious at a glance:

```suggestion
func (a *ACLToken) HasRoles(roleIDs []string) bool {
```

Reply:
Agreed and has been updated within commit: https://github.com/hashicorp/nomad/pull/14923/commits/3481a620406ba5b1f2c4d1c29047e0a9d82cef67

Thanks.